Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4148,3 +4150,45 @@ bool CLuaVehicleDefs::BlowVehicle(CClientEntity* entity, std::optional<bool> wit
 {
     return CStaticFunctionDefinitions::BlowVehicle(*entity, withExplosion);
 }
+
+std::variant<bool, std::vector<std::array<float, 3>>> CLuaVehicleDefs::GetVehicleEntryPoints(CClientVehicle* vehicle)
+{
+    if (CClientVehicleManager::GetMaxPassengerCount(vehicle->GetModel()) == 255)
+    {
+        return false;
+    }
+
+    std::vector<std::array<float, 3>> entryPoints;
+
+    for (unsigned int i = 0; i <= 3; i++)
```

Comment:
Use `i < 4` as the condition please, its really confusing.

Reply:
Interestingly enough, that's what I usually do. No idea why I didn't in this case. I did take this piece of code from an earlier PR I had worked on so I past me is to blame for this. Will change it!